lms/static/sass/multicourse/_dashboard.scss

```diff
@@ -1,1214 +1,1075 @@
 // lms - views - user/student dashboard
 // ====================
 
-// Table of Contents
-// * +Dashboard - Sidebar
-// * +Dashboard - Course Listing
-// * +Dashboard - Course Item
-// * +Misc - Uncategorized
-// * +Dashboard - Banner
-
-
-// +Dashboard - Sidebar
+// +Dashboard
 // ====================
 .dashboard {
   @include clearfix();
 
-  padding: ($baseline*2) 0 $baseline 0;
-
-  .wrapper-find-courses {
-    @include float(right);
-    @include margin-left(flex-gutter());
+  display: flex;
+  flex-direction: row;
+  width: 100%;
 
-    width: flex-grid(3);
-
-    .course-advertise {
-      @include clearfix();
+  // Contains main course card listings
+  .main-container {
+    @include padding($baseline*2, $baseline, $baseline, $baseline*2);
 
-      box-sizing: border-box;
-      padding: $baseline;
-      border: 1px solid $border-color-l3;
+    flex-grow: 8;
+    order: 1;
 
-      .advertise-message {
-        @include font-size(12);
+    .my-courses {
+      &:focus {
+        outline: none;
+      }
 
-        color: $gray-d4;
+      .wrapper-header-courses {
+        border-bottom: 4px solid $border-color-l4;
```

Comment:
Is there a Bootstrap color variable you can use instead of this one? I'd like to kill all these variants...